services/core-web/src/components/mine/NoticeOfWork/MineApplications.js

```diff
@@ -92,14 +91,6 @@ export class MineApplications extends Component {
     );
   };
 
-  onExpand = (expanded, record) =>
```

Comment:
Not necessary to write logic to handle row expansion, or have expandedRows state variables, _unless_ expansion is triggered by anything other than the user clicking the expand icon.